### PR TITLE
refactor: kill orphaned dependencies

### PR DESCRIPTION
**Pull request by kyranjamie**
> Try out this version of the Hiro Wallet - download [extension builds](https://github.com/blockstack/ux/actions/runs/1112201568).

Removed some unused files found with depcruiser

### CODE REVIEW

**Review by fbwoolf — APPROVED** [review]

File: src/components/powered-by.tsx [hunk 1]
```diff
@@ -0,0 +1,38 @@
+import React from 'react';
```
Comment (aulneau) [inline src/components/powered-by.tsx:1]: This file should be deleted, yeah?

File: src/pages/choose-account.tsx [hunk 2]
```diff
@@ -1,49 +0,0 @@
-import React, { memo, useCallback, useEffect } from 'react';
```
Comment (aulneau) [inline src/pages/choose-account.tsx]: seems misconfigured, this is 100% used unless something changed

Comment (aulneau) [reply src/pages/choose-account.tsx]: https://github.com/blockstack/stacks-wallet-web/blob/main/src/routes.tsx#L7

File: src/components/screen/index.tsx [hunk 3]
```diff
@@ -1,9 +0,0 @@
-export * from './screen';
```
Comment (aulneau) [inline src/components/screen/index.tsx:1]: https://github.com/blockstack/stacks-wallet-web/blob/main/src/pages/choose-account.tsx#L2

File: src/pages/choose-account.tsx (hunk 2 above)
Comment (kyranjamie) [reply src/pages/choose-account.tsx]: maybe I hadn't committed these changes yet? typecheck passes locally

**Review by aulneau — APPROVED** [review]
LGTM, shipit

**Comment by aulneau** [comment]
@kyranjamie seems there are some conflicts -- once those are good we should merge

**Comment by aulneau** [comment]
@kyranjamie would you be able to pick this back up?

**Pull request merged**